io/pci/pci_hotplug.py

```diff
@@ -43,14 +43,21 @@ def setUp(self):
         cmd = "uname -p"
         if 'ppc' not in process.system_output(cmd, ignore_status=True):
             self.skip("Processor is not ppc64")
+        if cpu._list_matches(cpu._get_cpu_info(), 'pSeries'):
+            for mdl in ['rpaphp', 'rpadlpar_io']:
+                if not linux_modules.module_is_loaded(mdl):
+                    linux_modules.load_module(mdl)
+        elif cpu._list_matches(cpu._get_cpuinfo(), 'PowerNV'):
+            if not linux_modules.module_is_loaded("pnv_php"):
+                linux_modules.load_module("pnv_php")
         self.return_code = 0
         self.device = self.params.get('pci_device', default=' ')
         if not os.path.isdir('/sys/bus/pci/devices/%s' % self.device):
             self.skip("PCI device given does not exist")
-        cmd = "cat /sys/bus/pci/devices/%s/devspec ; echo" % self.device
-        self.devspec = process.system_output(cmd, shell=True).strip('\n')
-        cmd = "cat /proc/device-tree/%s/ibm,loc-code ; echo" % self.devspec
-        self.slot = process.system_output(cmd, shell=True).strip('\0\n')
+        self.devspec = genio.read_file("/sys/bus/pci/devices/%s/devspec"
```

Comment:
can  `self.devspec ` local variable as it seems me used out side `setUp` please consider